src/spetlr/autoloader/autoloader_handle.py

```diff
@@ -0,0 +1,66 @@
+from pyspark.sql import DataFrame
+
+from spetlr.configurator.configurator import Configurator
+from spetlr.delta.delta_handle import DeltaHandleInvalidFormat
+from spetlr.exceptions import SparkVersionNotSupportedForSpetlrStreaming
+from spetlr.spark import Spark
+from spetlr.tables import TableHandle
+
+
+class AutoloaderHandle(TableHandle):
```

Comment:
I think the name of this class is a bit off. I think it can be confusing that the handle has `loader` in its name. It can be used for extraction as well. Also, the word `Auto` does not tell me much about what it "handles". Suggestion `FileHandle`

Reply:
I think you have an excellent point... I will try it out.